src/scss/pages/_pricing.scss

```diff
@@ -0,0 +1,86 @@
+.Pricing {
+    &-plans {
+        align-items: center;
+        display: flex;
+        height: 878px;
+        justify-content: center;
+        width: 100%;
+    }
+
+    &-container {
+        width: 1110px;
+    }
+
+    &-title {
+        align-items: center;
+        color: $button-color;
+        display: flex;
+        font-family: "Rajdhani", sans-serif;
+        font-size: 30px;
+        font-weight: 100;
+        justify-content: center;
+        margin-bottom: 85px;
+        width: 1110px;
+    }
+
+    &-box {
```

Comment:
Delete this and use the classes that bootstrap provides.

Reply:
done